Documentation/advanced-configuration.md

```diff
@@ -18,6 +19,29 @@ The Kubernetes based examples assume Rook OSD pods are in the `rook` namespace.
 If you run them in a different namespace, modify `kubectl -n rook [...]` to fit
 your situation.
 
+## Log Collection
```

Comment:
I would prefer an utility script like `getLog.sh` or something. But im ok with this for now.